src/vsr/replica.zig

```diff
@@ -483,16 +483,7 @@ pub fn ReplicaType(
         repair_messages_budget_grid: vsr.Budget,
 
         repair_messages_budget_journal: vsr.Budget,
-
-        // The next op between [commit_min + 1, op] to be requested during repair.
-        // Used to cycle through uncommitted prepares, to avoid requesting the same missing prepares
-        // repeatedly. Reset to commit_min + 1 when repair timeout is fired.
-        repair_prepare_op_next_uncommitted: u64 = 0,
-
-        // The next op between [op_repair_min, commit_min] to be requested during repair.
-        // Used to cycle through committed prepares, to avoid requesting the same missing prepares
-        // repeatedly. Reset to op_repair_min when repair timeout is fired.
-        repair_prepare_op_next_committed: u64 = 0,
+        repair_messages_inflight_prepare: std.AutoArrayHashMapUnmanaged(u64, void),
```

Comment:
Not entirely sure, but my gut feeling would be to raise a level of abstraction here: 

```zig
const PreparesRequested = struct {
    set: std.AAHMU(u64, void), 

    pub fn init(gpa: Allocator, capacity: u32)
    pub fn deinit()

    pub fn add(prepares_request: *PreparesRequested, header: vsr.Header) enum { added, already_present }
    pub fn remove(prepares_request: *PreparesRequested, header: vsr.Header) void;
    pub fn reset(prepares_request: *PreparesRequested) void; 
}
```


Maybe it is premature abstraction, but it feels like it might capture a real concept here?

(note: havent' read the diff yet, the API is what I _imagine_ we need here, but probably every name is wrong there). But I think using headers, rather than ops, is a good idea --- binds intent!